test/compress/issue-208.js

```diff
@@ -27,3 +27,28 @@ do_update_rhs: {
         MY_DEBUG += 0;
     }
 }
+
+mixed: {
+    options = {
+        global_defs: { DEBUG: 0 }
+    }
+    input: {
```

Comment:
I tried the following:
```js
global_defs: {
    DEBUG: 0,
    ENV: 1,
    FOO: 2,
}


const ENV = 3;
var FOO = 4;
f(ENV);
--FOO;
DEBUG = 1;
DEBUG++;
DEBUG += 1;
f(DEBUG);
x = DEBUG;
```
No new warnings. Do you wish to have warnings there? The example I quote in https://github.com/mishoo/UglifyJS2/pull/1467#issuecomment-277790228 would then give a warning, for instance.